recipes/dinopy/meta.yaml

```diff
@@ -7,8 +7,8 @@ source:
   sha256: 092323edddf48c9d3082dfaf934f5c3e4011c7e1248503cd352105e6e50bfcf5
 
 build:
-  skip: True # [py27]
-  number: 0
+  skip: True # [py27 or py36]
+  number: 2
```

Comment:
Why the increase to `2`, not just `1`?

Reply:
I guess I accidentally increased it twice. But instead of rebuilding now again, I think it is fine to keep it like this.

Reply:
Alright.